deliverables/README.md

```diff
@@ -1,68 +1,140 @@
 
+That way it is easy to distinguish the content we are editing rather than the design of the code.
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+What are classes and IDs (and how are they different)?
+ID's are unique throughout the code and allow you to label and edit one aspect while Classes are not unique and allow you to access and edit more than one portion of code at a time.
+
+What are elements?
+Elements are used to organize the components of code in a web page. They always have two tags, a beginning and end.
+
+What are tags?
+Tags are used to define the HTML and apply something to the content.
+
+What are attributes?
+There are many different kind of attributes that can go inside a tag to modify the current tag.
+
+What are forms?
+Forms are used within webpages to make the website more interactive and gather information, it takes some kind of information and sends it to some kind of database.
+
+What is a div?
+Div tag works as a container for HTML code and can be used to combine / label code.
+
+What are selectors?
+In CSS, selectors are patterns used to select the element(s) you want to style.
+
+What are properties?
+Code can have properties just like a dog has fur, a tail, and eye color.
+
+What are values?
+The variable name is the usual way to reference the stored value; this separation of name and content allows the name to be used independently of the exact information it represents. The identifier in computer source code can be bound to a value during run time, and the value of the variable may thus change during the course of program execution
+
+How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?
+The idea behind inheritance is relatively easy to understand. Elements inherit styles from their parent container. If you set the body tag to use color: red then the text for all elements inside the body will also be red unless otherwise specified.
+
+What are two CSS attributes you can change to push an element around on the page?
+An absolutely positioned element is always relative to the next positioned parent. If there isn't a parent element, the containment block is used instead. Values for left and top are relative to the upper-left corner of the next positioned element in the hierarchy. For example, to place an image at the top left corner of the document, set the attributes to 0.  Fixed positioning is also another attribute you can add to an element.
```

Comment:
Once again, link to source:  https://msdn.microsoft.com/en-us/library/ms533005%28v=vs.85%29.aspx